juliacalldeps.json

```diff
@@ -0,0 +1,11 @@
+{
+    "julia": "1.7.0",
+    "packages": {
+        "ARTime": {
+            "uuid": "9badb8af-59cb-4d48-a775-3f35cfcfd143",
+            "compat": "~0",
+            "dev": true,
+            "path": "/home/mark/ARTimeNAB.jl"
```

Comment:
When trying to reproduce the results following the [README](https://github.com/numenta/NAB/blob/adc84d7f5d7a4ef02a9e0a0968eafd35984fe926/nab/detectors/ARTime/README.md) instructions on a clean environment I got this error:
```
% python run.py -d ARTime --detect --optimize --score --normalize --skipConfirmation

Running detection step
Querying Julia versions from https://julialang-s3.julialang.org/bin/versions.json
Downloading Julia from https://julialang-s3.julialang.org/bin/mac/x64/1.7/julia-1.7.2-mac64.dmg
  downloaded 69.2 MB of 99.3 MB
  download complete
Verifying download
Installing Julia 1.7.2 to ~/.julia/pythoncall/julia-1.7.2
  Installing known registries into `~/.julia`
┌ Error: Error loading PythonCall.jl
│   err = Dev path `/home/mark/ARTimeNAB.jl` does not exist.
└ @ Main none:9
Traceback (most recent call last):
  File "run.py", line 204, in <module>
    main(args)
  File "run.py", line 69, in main
    runner.detect(detectorConstructors)
  File "~/NAB/nab/runner.py", line 122, in detect
    probationaryPercent=self.probationaryPercent),
  File "~/NAB/nab/detectors/ARTime/ARTime_detector.py", line 39, in __init__
    from juliacall import Main as jl
  File "~/miniconda3/envs/NAB/lib/python3.6/site-packages/juliacall/__init__.py", line 4, in <module>
    from . import init
  File "~/miniconda3/envs/NAB/lib/python3.6/site-packages/juliacall/init.py", line 155, in <module>
    raise Exception('PythonCall.jl did not start properly')
Exception: PythonCall.jl did not start properly
```
I think it is because this dependency. 
Is it possible to use the dependency directly from github instead of local path?